server/backend/src/cameras/camera-auth.guard.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  ForbiddenException,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Request } from 'express';
+import { CamerasService } from './cameras.service';
+import { verify } from 'crypto';
+import { stringToBytes } from '../util';
+
+@Injectable()
+export class CameraAuthGuard implements CanActivate {
+  constructor(private cameraService: CamerasService) {}
+
+  public async canActivate(context: ExecutionContext): Promise<boolean> {
+    const request = context.switchToHttp().getRequest<Request>();
+    const id = request.params['id'];
+    if (id === undefined) {
+      return true;
```

Comment:
Why do we return true here?

Reply:
I was thinking that we might have camera endpoints that aren't associated with a specific camera, and in that case this guard shouldn't deny access, but I don't think any of our camera endpoints will be unrestricted so this return should probably give back `false`.

Reply:
resolved in [`4952b1a`](https://github.com/tucker-moore/human-detector/pull/31/commits/4952b1acf40e124d45f73d4f084785770aa70d49)